vermouth/processors/do_links.py

```diff
@@ -143,25 +144,25 @@ def match_order(order1, resid1, order2, resid2):
     (order1 argument), while the columns correspond to the order at the right
     of it (order2 argument).
 
-    +----+---+----+---+----+---+---+---+----+
-    |    | > | >> | < | << | n | 0 | * | ** |
-    +----+---+----+---+----+---+---+---+----+
-    | >  | = | <  | > | >  | ! | > | ! | !  |
-    +----+---+----+---+----+---+---+---+----+
-    | >> | > | =  | > | >  | ! | > | ! | !  |
-    +----+---+----+---+----+---+---+---+----+
-    | <  | < | <  | = | >  | ! | < | ! | !  |
-    +----+---+----+---+----+---+---+---+----+
-    | << | < | <  | < | =  | ! | < | ! | !  |
-    +----+---+----+---+----+---+---+---+----+
-    | n  | ! | !  | ! | !  | ? | ? | ! | !  |
-    +----+---+----+---+----+---+---+---+----+
-    | 0  | < | <  | > | >  | ? | = | / | /  |
-    +----+---+----+---+----+---+---+---+----+
-    | *  | ! | !  | ! | !  | ! | / | = | /  |
-    +----+---+----+---+----+---+---+---+----+
-    | ** | ! | !  | ! | !  | ! | / | / | =  |
-    +----+---+----+---+----+---+---+---+----+
+    +-----+---+----+---+----+---+---+----+-----+
+    |     | > | >> | < | << | n | 0 | \* | \** |
```

Comment:
I guess rest tried to interpret the stars?

Reply:
Jup